### ISSUE TITLE
Procedural Visual Novel

### ISSUE THREAD

**Issue by dariusk** [dariusk/NaNoGenMo-2015#2]
This year I'm going to try and generate a 50,000 word [visual novel](https://en.wikipedia.org/wiki/Visual_novel).

**Comment by MaxBittker** [comment]
what are your ideas for the visual part of this visual novel? have you ever clicked around drawr? it's like pinterest+deviant art  http://drawr.net/

**Comment by MichaelPaulukonis** [comment]
Are you going to actually post your code this year? &lt;/dubious>

**Comment by dariusk** [comment]
I didn't finish a novel last year so I didn't post any code.

**Comment by ikarth** [comment]
Solved the problem of posting the result on the web?

**Comment by enkiv2** [comment]
I'm glad you're trying this out this year. Very disappointed that you gave up on it last year (and disappointed that my ~2 attempts at generated VNs were largely unsuccessful)

**Comment by dariusk** [comment]
@ikarth Yeah, there's stuff like [TyranoScript](http://www.evanburchard.com/tyranoscript/) now!
